swap/log.go

```diff
@@ -0,0 +1,114 @@
+package swap
+
+import (
+	l "github.com/ethereum/go-ethereum/log"
```

Comment:
in this case i also think we should use `log` instead of `l`, for clarity's sake